ui/src/authors/components/PublicationsSelect.jsx

```diff
@@ -19,10 +19,12 @@ function PublicationsSelect({
     }
     if (claimed) {
       onSelectClaimedPapers(event);
-    } else if (!claimed && canClaim) {
+    }
+    if (!claimed) {
```

Comment:
are you sure we should remove the `canClaim`?

Reply:
after talking to @MJedr I know I can't do it like this. I'll convert the PR into draft and try to do it another way